libs/core/src/lib/select/select.component.html

```diff
@@ -1,31 +1,24 @@
-<fd-popover [(isOpen)]="isOpen"
-            (isOpenChange)="isOpenChangeHandle($event)"
-            [options]="popperOptions"
-            [fillControlMode]="fillControlMode"
-            [appendTo]="appendTo"
-            class="fd-select-popover-custom">
+<fd-popover [(isOpen)]="isOpen" (isOpenChange)="isOpenChangeHandle($event)" [options]="popperOptions"
+    [fillControlMode]="fillControlMode" [appendTo]="appendTo" class="fd-select-popover-custom">
     <fd-popover-control>
         <ng-container *ngIf="triggerTemplate">
             <ng-container *ngTemplateOutlet="triggerTemplate; context: {$implicit: this}"></ng-container>
         </ng-container>
         <ng-container *ngIf="!triggerTemplate">
-            <button class="fd-select-button-custom"
-                    fd-button
-                    aria-haspopup="true"
-                    [ngClass]="{'fd-button--compact': compact}"
-                    [attr.aria-expanded]="isOpen"
-                    [disabled]="disabled">
-                <span class="fd-select-text-custom">{{triggerValue}}</span>
+            <button fd-button class="fd-select-button-custom" aria-haspopup="true" [ngClass]="{
+                    'fd-select-button-custom--loading': loading,
+                    'fd-button--compact': compact
```

Comment:
Maybe let's change it to `[compact]="compact", it's been possible since we've got `fd-button` everywhere

Reply:
changed